cluster/node.go

```diff
@@ -267,6 +269,14 @@ func (n *Node) ReservedCpus() int64 {
 	return r
 }
 
+func (n *Node) OverCommitedMemory() int64 {
```

Comment:
How about "unreserved"? I really dislike "available" for the confusion you mentioned.